tachyon/zk/base/entities/prover_base.h

```diff
@@ -44,13 +44,15 @@ class ProverBase : public Entity<PCS> {
   [[nodiscard]] bool Commit(const Poly& poly) {
```

Comment:
Please add a comment for usage

```suggestion
   // Commit |poly|. Note that if |this->pcs_.batch_mode()| is true, it doesn't write a commitment to proof.
   [[nodiscard]] bool Commit(const Poly& poly) {
```